tests/test_browse.py

```diff
@@ -33,6 +34,34 @@ def test_home(self):
         self.assertFalse(html.find('h2', text='Test'),
                          "'Test' group should not be shown on homepage")
 
+    def test_tb(self):
+        """Test the /tb/<arxiv_id> page."""
+        rv = self.app.get('/tb/0808.4142')
+        self.assertEqual(rv.status_code, 200)
+
+        rv = self.app.get('/tb/1901.99999')
+        self.assertEqual(rv.status_code, 404)
+
+        rv = self.app.get('/tb/')
+        self.assertEqual(rv.status_code, 404)
+
+        rv = self.app.get('/tb/foo')
+        self.assertEqual(rv.status_code, 404)
+
+    def test_tb(self):
```

Comment:
Is this the same method name as the previous method on line 37? Is that Okay?

Reply:
Ooops, should have been called `test_tb_recent()`--fixed.